app/helpers/fhir_helper.rb

```diff
@@ -265,8 +338,8 @@ def language_coding(language)
   end
 
   # Helper to understand a boolean extension
-  def from_bool_extension(patient, extension_id)
-    patient&.extension&.find { |e| e.url.include?(extension_id) }&.valueBoolean
+  def from_bool_extension_false_default(patient, extension_id)
+    { value: patient&.extension&.find { |e| e.url.include?(extension_id) }&.valueBoolean || false, path: patient_bool_ext_path(extension_id) }
   end
 
   def to_bool_extension(value, extension_id)
```

Comment:
Oh and there are also a few places where `http://hl7.org/fhir/StructureDefinition/data-absent-reason` is used instead of `DATA_ABSENT_URL` that could also be replaced.=

Reply:
Done https://github.com/SaraAlert/SaraAlert/pull/743/commits/cacd404f27bcb86cb4efd6086fe9c394f1a56764